test/integration/spec/localtrackpublication.js

```diff
@@ -515,7 +515,7 @@ describe('LocalTrackPublication', function() {
     [PRIORITY_LOW, PRIORITY_STANDARD, PRIORITY_HIGH].forEach(beforePriority => {
       [PRIORITY_LOW, PRIORITY_STANDARD, PRIORITY_HIGH].forEach(afterPriority => {
         const expectNotification = beforePriority !== afterPriority;
-        it(`VMS-2231:subscriber ${expectNotification ? 'gets notified' : 'does not get notified'} when publisher changes priority: ${beforePriority} => ${afterPriority}`, async () => {
```

Comment:
I see that you are reusing the same bug number. Lets file separate bugs for each test that is unstable.